src/SourceMapUtils.h

```diff
@@ -0,0 +1,44 @@
+//
```

Comment:
I'm wondering why this code is not buried somewhere in an anonymous namespace in a source file.

Reply:
Is is part of code whi was used in reporting. I did not wrote it jost move from another file to public space.
Reason to be publish it, is it is used in two parts - first is `reporting.cc` and second one is used in annotation line/column info